package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "directus-extension-group-tabs-interface",
 	"icon": "extension",
-	"version": "1.2.2",
+	"version": "1.3.0",
```

Comment:
The version number stays the same, semantic-release-bot handles the increase for us.